gflownet/envs/crystals/composition.py

```diff
@@ -367,3 +367,14 @@ def _can_produce_neutral_charge(self, state: Optional[List[int]] = None) -> bool
         ]
 
         return any(poss_charge_sum)
+
+    def get_diff_elem_number(self, state=None):
+        if state is None:
+            state = self.state
+        return (np.array(state) != 0).sum()
+
+    def get_atoms_number(self, state=None):
```

Comment:
Just a matter of taste, but I would prefer a method name like:
- `get_number_of_atoms` or
- `get_n_atoms`

Reply:
Ok, I've changed that